ror-with-kibana-reverse-proxy-demo/utils/collect-info-about-ror-es-kbn.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+echo "Preparing Elasticsearch & Kibana with ROR environment ..."
+
+if ! command -v jq > /dev/null; then
+  ./collect-info-about-ror-es-kbn-without-hints.sh
+else
+  ./utils/collect-info-about-ror-es-kbn-with-hints.sh || {
+    if [[ $? -eq 28 || $? -eq 128 ]]; then
+      ./utils/collect-info-about-ror-es-kbn-without-hints.sh
+    else
+      exit $?
```

Shellcheck (0.11.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)